finder-app/Makefile

```diff
@@ -0,0 +1,15 @@
+TARGET_FILE = writer
+#CROSS_COMPILE = aarch64-none-linux-gnu-
+#if CROSS_COMPILE SPECIFIED IN COMMAND LINE
+
+ifdef CROSS_COMPILE
```

Comment:
@ayswariya088 see a better way to do this in the Embedded System Toolchains video.  You shouldn't need the `ifdef`.